modules/ROOT/pages/dw-timer-types.adoc

```diff
@@ -4,15 +4,15 @@ include::_attributes.adoc[]
 endif::[]
 
 |===
-| Type | Definition | Description
+a| Type a| Definition a| Description
 
-| DurationMeasurement
-| `type DurationMeasurement = { time: Number, result: T }`
-| Represents a time taken by a function call.
+a| DurationMeasurement
+a| `type DurationMeasurement = { time: Number, result: T }`
+a| Represents a time taken by a function call.
 
 
-| TimeMeasurement
-| `type TimeMeasurement = { start: DateTime, result: T, end: DateTime }`
-| Represents a start/end time measurement.
+a| TimeMeasurement
+a| `type TimeMeasurement = { start: DateTime, result: T, end: DateTime }`
+a| Represents a start/end time measurement.
```

Comment:
What is a time measurement? the amount of time including/excluding start and end datetime?